### ISSUE TITLE
Jare's Proposal Seven for Yield-Generating Opportunity: Cut Percent Fees to Folks with Ideas, Dev Shops 

### ISSUE THREAD

**Issue by DunnCreativeSS** [Stakedllc/robo-advisor-yield#9]
## Why 
There's a 'bounty' to create or implement proposals: https://github.com/Stakedllc/robo-advisor-yield/wiki/Requests-for-Contracts - there'd be far more movement if this were a residual % of the fees generated by RAY

## Solution
Re-word doc to include (or offer as an alternative) residual %s

## Considerations
Depends on business requirements...

## Additional Info
I was under the impression should these ideas be implemented they'd generate residual revenues :) This would be ideal.

**Comment by dpurhar27** [comment]
Yes, this is good point. We're mid-forming a proper framework for proposals for RAY. Generally, we're starting to put together a trust-less system or "governance" that does this. The core goal is to align economic incentives so contributors earn as RAY earns as you suggest here.

I'll notify you when this discussion is posted publicly, it should be in the next few weeks. It will be great to have someone interested in contributing to RAY, as you are, join in and let us know your thoughts and feedback.
